runtime-sdk-macros/src/test_utils.rs

```diff
@@ -0,0 +1,64 @@
+pub fn rustfmt(code: &str) -> String {
+    use std::{
+        io::Write,
+        process::{Command, Stdio},
+    };
+
+    let mut cp = Command::new("rustfmt")
```

Comment:
Using rustfmt directly (as was done before) doesn't work?

Reply:
It's an old, deprecated one; the crate says so. Including the nightly one as a lib pulls in all sorts of auto-published rust internals.